src/Files.App/Helpers/ThemeHelper.cs

```diff
@@ -62,24 +63,19 @@ public static void Initialize()
 		private static async void UiSettings_ColorValuesChanged(UISettings sender, object args)
 		{
 			// Make sure we have a reference to our window so we dispatch a UI change
-			if (currentApplicationWindow is not null)
-			{
-				// Dispatch on UI thread so that we have a current appbar to access and change
-				await currentApplicationWindow.DispatcherQueue.EnqueueAsync(() =>
-				{
-					ApplyTheme();
-				});
-			}
+			if (currentApplicationWindow is null)
+				return;
```

Comment:
If `currentApplicationWindow` is `null` here, shouldn't we try initialize again?

Reply:
Wouldn't we get stuck in a loop?

Reply:
We should try one time every the event raised.

Reply:
I'm curious what you have in mind, would you add a Boolean to keep track?

Reply:
I have like this in mind.
```csharp
		private static async void UiSettings_ColorValuesChanged(UISettings sender, object args)
		{
			// Make sure we have a reference to our window so we dispatch a UI change
			if (currentApplicationWindow is null)
			{
				currentApplicationWindow = App.Window;

				if (currentApplicationWindow is not null)
					titleBar = App.GetAppWindow(currentApplicationWindow).TitleBar;
				else
					return;
			}

			// Dispatch on UI thread so that we have a current appbar to access and change
			await currentApplicationWindow.DispatcherQueue.EnqueueAsync(ApplyTheme);
		}
```

Reply:
@hishitetsu can you open a PR?